src/Sarif.Viewer.VisualStudio/Converters/ImportanceToForegroundConverter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.Sarif.Viewer.Models;
+using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Data;
+
+namespace Microsoft.Sarif.Viewer.Converters
+{
+    class ImportanceToForegroundConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            var node = value as CallTreeNode;
+            if (node != null)
+            {
+                if (node.Location.Importance == AnnotatedCodeLocationImportance.Unimportant)
+                {
+                    return Color.Gray;
+                }
+                else
+                {
+                    return Color.Black;
+                }
```

Comment:
You could use the conditional operator here:

return node.Location.Importance == AnnotatedCodeLocationImportance.Unimportant
    ? Color.Gray
    : Color.Black